hivemind/optim/collaborative.py

```diff
@@ -344,38 +371,38 @@ def accumulated_grads(self) -> Iterator[torch.Tensor]:
         """local gradient accumulators"""
         if self.reuse_grad_buffers:
             yield from self._grad_buffers()
-        elif self._grads is None:
-            with torch.no_grad():
+            return
+        else:
```

Comment:
I respectfully disagree here: it does reduce indentation, but:
- the code in the else clause is quite simple as it is
- crucially, last time the code was wrong because i've accidentally removed return and didn't think about the ramifications

If you do insist, please state that explicitly, i'll remove the else clause anyway

Reply:
In this case, I do not insist :)

Reply:
I would say that `else:` after `return` is obviously redundant and seems like a bug, so I insist on choosing one of the two options:

1. `return` without `else:` (preferred since it implies less indentation, more clarity)
2. `else:` without `return`